web-app/src/app/screens/Feed.tsx

```diff
@@ -0,0 +1,603 @@
+                                '0'
+                              } Info Notices`}
+                              color='primary'
+                              variant='outlined'
+                            />
+                          </TableCell>
+                        </TableRow>
+                        <TableRow>
+                          <TableCell>
+                            <span
+                              style={{
+                                display: 'flex',
+                              }}
+                            >
+                              <a
+                                href={`${latestDataset?.validation_report?.url_html}`}
+                                target='_blank'
+                                rel='noreferrer'
+                              >
+                                Open Full Report
+                              </a>
+                              <OpenInNewOutlined />
+                            </span>
+                          </TableCell>
+                          <TableCell>
+                            <span
+                              style={{
+                                display: 'flex',
+                              }}
+                            >
+                              <a
+                                href={`${latestDataset?.validation_report?.url_json}`}
+                                target='_blank'
+                                rel='noreferrer'
+                              >
+                                Open JSON Report
+                              </a>
+                              <OpenInNewOutlined />
+                            </span>
+                          </TableCell>
+                        </TableRow>
+                      </TableContainer>
+                    </ContentBox>
+                  </Grid>
+                )}
+                {feed?.data_type === 'gtfs' && (
+                  <Grid item xs={12}>
+                    <ContentBox
+                      width={{ xs: '100%', md: '50%' }}
+                      title={'Features List'}
+                      outlineColor={colors.indigo[500]}
+                    >
+                      <TableContainer>
+                        {latestDataset?.validation_report?.features !==
+                          undefined &&
+                          latestDataset?.validation_report?.features?.length >
+                            0 && (
+                            <TableRow>
+                              <TableCell>
+                                <b>Feature</b>
+                              </TableCell>
+                            </TableRow>
+                          )}
+                        {latestDataset?.validation_report?.features?.map(
+                          (v) => (
+                            <TableRow key={v}>
+                              <TableCell>{v}</TableCell>
```

Comment:
format feature name using `DATASET_FEATURES_FILES_MAPPING`  + add second column of "File or Field Associated"

Reply:
Based on my conversation with @emmambd, I'm under the impression we are omitting "File or Field Associated"

Reply:
Correct, but the feature names themselves will be used

Reply:
For `mdb-123` does this look correct? 
![image](https://github.com/MobilityData/mobility-feed-api/assets/444765/650c6bd8-64de-4c82-aae7-23b5842bd872)